### PR TITLE
Improve readme and documentation

### PR DESCRIPTION
**Pull request by jamesrichford**
# Description
Moving over to a wiki approach makes it more readable and neatens up the readme. @Jameskmonger any thoughts?

# Checklist

- [x] I am an awesome developer and proud of my code
- [x] I added / updated / removed relevant unit or integration tests to prove my change works
- [x] I ran all tests using ```npm test``` to make sure everything else still works
- [x] I ran ```npm run review``` to ensure the code adheres to the repository standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,792 +32,79 @@
...
+If at any time things are unclear or you think there may be something going wrong feel free to [raise an issue](https://github.com/alsatian-test/alsatian/issues/new) and we'll be glad to get back to you with a solution quickly.
 
-```typescript
-import { ExtendedExpect as Expect } from "./your/extended-expect/location";
-import { TestFixture, Test } from "alsatian";
+## Contributing
 
-@TestFixture("color tests")
-export default class ColorTestFixture {
+We're always glad to have help out with Alsatian, check out the [guidelines](https://github.com/alsatian-test/alsatian/blob/master/CONTRIBUTING.md)]
```
Comment (Nysosis) [inline README.md]: Looks like there's a trailing `]` at the end of this line

File: README.md [hunk 2]
```diff
@@ -32,792 +32,79 @@
...
+Otherwise if you're more of a JavaScript kinda person, have a gander at our [JavaScript Quick Start](https://github.com/alsatian-test/alsatian/wiki/javascript-setup)
 
-// setup the output
-testRunner.outputStream
-          // this will use alsatian's default output if you remove this
-          // you'll get TAP or you can add your favourite TAP reporter in it's place
-          .pipe(TapBark.create().getPipeable()) 
-          // pipe to the console
-          .pipe(process.stdout);
+Full documentation can be found [here](https://github.com/alsatian-test/alsatian/wiki/) the examples are usually in TypeScript but should be pretty much the same code for JavaScript (just lose the access modifiers and types).
```
Comment (Nysosis) [inline README.md]: Maybe change here to on our/the wiki. Reads nicer (in my opinion), and accessibility guidelines tend to say don't use here as link text

**Comment by Nysosis** [comment]
Looking good, though I think it's worth keeping the CLI options in the readme, its one of the most useful things to know for any library.

There's also a reference to using nyc for code coverage, that could do with either having a page on the wiki how to do that, or a link off to the nyc repo/package.

**Comment by jamesrichford** [comment]
@Nysosis ace feedback :) made some changes as you suggested

**Pull request merged**